tests/wpt/metadata/MANIFEST.json

```diff
@@ -527055,42 +527055,6 @@
       }
      ]
     ],
-    "send-data-readablestream.any.js": [
```

Comment:
We should ensure there is a test that verifies the behaviour of xhr send when a ReadableStream argument is passed. This will make it clear on wpt.fyi if browsers have actually converged on the spec in the future.